Makefile

```diff
@@ -0,0 +1,12 @@
+default: help
+
+.PHONY : help
+help:  ## Show this help
+	@echo "$(BOLD)FUN AWS Makefile$(RESET)"
+	@echo "Please use 'make $(BOLD)target$(RESET)' where $(BOLD)target$(RESET) is one of:"
+	@grep -h ':\s\+##' Makefile | column -tn -s# | awk -F ":" '{ print "  $(BOLD)" $$1 "$(RESET)" $$2 }'
+
```

Comment:
You can add a `env.d/local` rule that copy the environment file template and a bootstrap rule that depends on the previous rule and `build`?